tests/installation/iso_checksum.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify data integrity of the iso image by comparing checksum.
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use Digest::file 'digest_file_hex';
+
+sub run {
+    my $self = shift;
+
+    my $iso      = get_required_var("ISO");
+    my $checksum = get_required_var("CHECKSUM_ISO");
+
+    my $digest = digest_file_hex($iso, "SHA-256");
+    diag "ISO checksum on the worker side: $digest";
+    ($checksum eq $digest) ? diag 'ok' : record_info('Failed', 'ISO checksum failed', result => 'fail');
```

Comment:
maybe you could include the expected and the actual checksum (and algo) in the `record_info`

Reply:
yes, right, better mentioning the algorithm and both things being compared. I put some extra spaces after ":" to see in the logs both aligned so it is easy to check it.